editor/inspector/contributions/node.js

```diff
@@ -90,15 +93,15 @@ exports.listeners = {
             }
         } catch (error) {
             console.error(error);
+        } finally {
+            Editor.Message.send('scene', 'snapshot');
```

Comment:
move snapshot from confirm-dump to here , would get correct snapshot data when have multiple selections